compass_metrics_model/metrics_model_lab.py

```diff
@@ -1 +1,205 @@
+import logging
+from datetime import datetime, timedelta
+from elasticsearch.exceptions import NotFoundError
+from grimoirelab_toolkit.datetime import (datetime_utcnow,
+                                          str_to_datetime)
+from grimoire_elk.enriched.utils import get_time_diff_days
+from .metrics_model import (MetricsModel,
+                            MAX_BULK_UPDATE_SIZE,
+                            check_times_has_overlap,
+                            create_release_index,
+                            get_medium)
+from .utils import (get_uuid, get_date_list)
+from .utils_lab import (starter_project_health, starter_project_health_decay)
 
+logger = logging.getLogger(__name__)
+
+def is_bot_by_author_name(*author_name):
```

Comment:
1. This part should not hard code here, prefer adding an extra config file to record all bot account. 
2. Except for full name of bot, also need to consider pattern, like **bot, to filter out bot.